pkg/beacon/relay/registry/marshalling_test.go

```diff
@@ -17,6 +17,9 @@ func TestMembershipRoundtrip(t *testing.T) {
 		group.MemberIndex(2),
 		new(bn256.G2).ScalarBaseMult(big.NewInt(10)),
 		big.NewInt(1),
+		map[group.MemberIndex]*bn256.G2{
+			group.MemberIndex(1): new(bn256.G2).ScalarBaseMult(big.NewInt(10)),
```

Comment:
Can we add one more entry to this map? In all real-world cases, we should have > 1 entry in this map.